developer-docs-site/docs/nodes/validator-node/operator/delegation-pool-operations.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Delegation Pool Operations"
+slug: "delegation-pool-operations"
+---
+
+# Delegation Pool Operations
+
+> Beta: This documentation is in experimental, beta mode. Supply feedback by [requesting document changes](../../../community/site-updates.md#request-docs-changes).
+
+Validator operators should follow these instructions to carry out delegation pool operations for [staking](../../../concepts/staking.md). You may delegate as little as 10 APT. Note that your validator will become part of the *Active Validator Set* only when the delegation pool satisfies the minimum cumulative [staking requirement of 1 million APT](./staking-pool-operations.md).
+
+Once the delegation pool attains 1M APT, the pool's owner who initiates the delegation pool should set an operator for the pool via the `set_operator` function described in the [Pool owner operations](#pool-owner-operations) section. The operator should then start their own Aptos node, as it is a best practice to have a different account for owner and operator. The operator should now [join in the active set of validators](./staking-pool-operations/#joining-validator-set).
+
+The operator address will receive the pool commission that was set at the initialization of the delegation pool and will act as a normal Delegation Pool account that is able to do all of the operations described in [Delegation pool operations](#delegation-pool-operations).
+
+It is not mandatory to operate an Aptos node personally to become a part of the active validators set. Instead, one can designate an individual with an Aptos node as the operator and associate the pool with their node. By doing so, the new operator will be able to successfully join the active validators set.
```

Comment:
This doesn't seem particularly important for delegation as it's generally the case for staking in Aptos

Reply:
Is it useful?

Reply:
I don't think so